test/unit/BondingManager.js

```diff
@@ -1132,14 +1132,15 @@ contract("BondingManager", accounts => {
         beforeEach(async () => {
             await fixture.roundsManager.setMockBool(functionSig("currentRoundInitialized()"), true)
             await fixture.roundsManager.setMockBool(functionSig("currentRoundLocked()"), false)
-            await fixture.roundsManager.setMockUint256(functionSig("currentRound()"), currentRound)
+            await fixture.roundsManager.setMockUint256(functionSig("currentRound()"), currentRound - 1)
```

Comment:
What is the reason for these additional test setup changes?

Reply:
otherwise the test for `"should fail for invalid unbonding lock ID"` would revert because the caller isn't bonded, rather than an invalid unbonding lock ID.

Reply:
Hm that's strange. In the original test setup, `delegator` should still be `Bonded` in `currentRound + 1` after it calls `unbond()` so the test for an invalid unbonding lock ID should pass. Tests pass for me locally when using the old test setup.

Reply:
I'll doublecheck

Reply:
Yeah works with the old set-up not sure what went wrong then. 

Fixed in 2eb6a0f